mungegithub/options/options.go

```diff
@@ -456,12 +457,29 @@ func toString(optType optionType, val interface{}) string {
 	}
 }
 
+func (o *Options) keysSortedAndWidth() ([]string, int) {
+	keys := make([]string, len(o.options))
+	width := 0
+	i := 0
+	for key, _ := range o.options {
+		keys[i] = key
+		if len(key) > width {
+			width = len(key)
+		}
+		i++
+	}
+	sort.Strings(keys)
+	return keys, width
+}
+
 func (o *Options) Descriptions() string {
 	var buf bytes.Buffer
 	fmt.Fprint(&buf, "The below options are available. They are listed in the format 'option: (default value) \"Description\"'.\n")
-	for key, opt := range o.options {
-		if opt.optType != typeUnknown {
-			fmt.Fprintf(&buf, "%s: (%s) %q\n", key, toString(opt.optType, opt.defaultVal), opt.description)
+	sorted, width := o.keysSortedAndWidth()
+	width++
+	for _, key := range sorted {
+		if opt := o.options[key]; opt.optType != typeUnknown {
+			fmt.Fprintf(&buf, "%-[2]*[1]s (%[3]s) %[4]q\n", key+":", width, toString(opt.optType, opt.defaultVal), opt.description)
```

Comment:
Fixed. I thought you had to use index specifiers to use the `*` formatting operator.